test/unit/base.test.js

```diff
@@ -2011,6 +2011,61 @@ describe('INTERFACE FUNCTION TESTS ', function () {
       });
     });
   });
+
+  describe('The converter must identify valid OA3 & 3.1 specifications', function () {
+    var pathPrefix = VALID_OPENAPI_PATH,
+      sampleSpecs = fs.readdirSync(path.join(__dirname, pathPrefix));
+
+    sampleSpecs.map((sample) => {
+      var specPath = path.join(__dirname, pathPrefix, sample);
+
+      it(specPath + ' is valid ', function(done) {
+        var openapi = fs.readFileSync(specPath, 'utf8'),
+          validationResult = Converter.validate({ type: 'string', data: openapi });
+
+        expect(validationResult.result).to.equal(true);
+        expect(validationResult.specificationVersion).to.be.oneOf(['3.0.x', '3.1.x']);
```

Comment:
@SahilChoudhary22 Don't we have any ways to differentiate this? i.e. specifically check 3.0 or 3.1 and not any one of it.

Reply:
@VShingala The `VALID_OPENAPI_PATH` contains specifications of both `3.0.x` and `3.1.x` versions, moving them might affect the existing test cases